src/en/news/blog/2022/newsletter-april/index.md

```diff
@@ -59,8 +59,7 @@ on the schedule as things progress.
 
 ## Welcoming New Foundation Members
 
-We are pleased to welcome two new Ceph Foundation members: [Arvan
-Cloud](https://www.arvancloud.com) and [Seagate](https://seagate.com)!
+We are pleased to welcome a new Ceph Foundation member, [Seagate](https://seagate.com)!
```

Comment:
It might not make sense to scrub them from the history... they did join in April.

Reply:
Ok, I reverted the change to this file.